src/app/common/services/unit-service.coffee

```diff
@@ -318,6 +318,45 @@ angular.module("doubtfire.common.services.units", [])
       else
         'No Tutorial'
 
+    student.calcTopTasks = () ->
+      #
+      # sort tasks by start date
+      #
+      sortedTasks = _.sortBy(_.sortBy(_.filter(student.tasks, (task) -> _.includes taskService.validTopTask, task.status), 'definition.seq'), 'definition.start_date')
+
+      currentWeight = 0
+
+      overdueTasks = _.filter sortedTasks, (task) ->
+        taskService.daysPastTargetDate(task) > 0
```

Comment:
Adjust this to use `task.daysPastTargetDate()`. You can add this if it does not exist.

Reply:
Done.

Reply:
Push up the changes so that the PR updates.

Reply:
Done.